src/zkevm_specs/util/arithmetic.py

```diff
@@ -1,10 +1,29 @@
 from __future__ import annotations
-from typing import Protocol, Sequence, Type, TypeVar, Union
-from functools import reduce
+from typing import Sequence, Protocol, Type, TypeVar, Union
 from py_ecc import bn128
 from py_ecc.utils import prime_field_inv
 
 
+def linear_combine(le_bytes: Sequence[Union[int, FQ]], base: FQ, range_check: bool = True) -> FQ:
```

Comment:
Maybe don't use the naming `bytes` for the variable, because input values higher than 255 when `range_check = False` are also valid?

Reply:
sounds good. I'll go with `seq`

Reply:
Addressed in [a375590](https://github.com/privacy-scaling-explorations/zkevm-specs/pull/240/commits/a375590df25dd9318f4b34d676096adbde15e235)